wiki/Community/Forum/Off-Topic/en.md

```diff
@@ -1,9 +1,92 @@
+
+
+### Exile
+
+Exile is an unofficial moderation concept introduced by [Ephemeral](https://osu.ppy.sh/users/102335) in [the grand edict thread](https://osu.ppy.sh/community/forums/topics/604424) in response to people repeatedly spamming the subforum and creating otherwise annoying content. The exile system is enforced by moderation by removing any thread, and sometimes posts, by the person who is exiled. It serves as an indicator that the person is on the verge of being heavily sanctioned by more official means of moderation.
+
+### Denizen
+
+Since the off-topic subforum is composed of mostly the same regulars chatting among themselves in threads, everyone comes to know one another other. Those who stick around long enough to be considered regulars are called "denizen". While this term was first used by [Ephemeral](https://osu.ppy.sh/users/102335) in [this thread](https://osu.ppy.sh/community/forums/posts/1607086), it was not adopted by regulars until its popularization by the [OT!News thread series](https://osu.ppy.sh/community/forums/topics/577518).
+
+### OT Lore
+
+OT Lore is an effort started by [abraker](https://osu.ppy.sh/users/4635891) and perpetuated by other denizens to create a story narrative consisting out of various threads people in OT create. The culmination of this effort was compiled into the [OT!Chronicles](https://osu.ppy.sh/community/forums/posts/6230570), and shaped a time period of OT which consisted of many improvised roleplay related threads.
+
+### Thread Derails
+
+Thread derailment is common in OT due to the definition of its nature - "Off-topic". This means a thread created with a topic in mind cannot be expected to remain on that topic.
+
+## Notable threads
+
+Despite majority of OT being nonsense and misc content of unimportance, there are some notable mentions for people unfamiliar with the subforum to check out.
+
+### OT!News series
+
+This series of threads is a great way to get a sense of the highs and lows of OT without seeing most of the nonsense.
+
+https://osu.ppy.sh/community/forums/posts/5942617
+
+### OT COURT II: A-Breaking Laboratory & R.O.S.Q. vs Asht. Co
+
+OT Court is OT Lore material involving a detailed imitation of an actual trial. Its organization was improvised on the spot and, against all odds, it successfully lasted from beginning to end without lost of interest or being derailed.
+
+https://osu.ppy.sh/community/forums/posts/6956061
+
+### Tad the Tadpole
+
+This is a joke thread turned in OT Lore material. It is a great example of how denizens can create jokes threads about each other and then proceed to build narrative off the resulting interactions. It also demonstrate the unpredictable nature of how threads evolve and the nonsensical randomness that may come out of it.
```

Comment:
Feels weird using "that has since" because it did not turn into OT Lore material after the thread ran its course, but rather  while people were still posting in it. I'm opting to use "that has developed" instead.